src/inject/settings/startBoot.js

```diff
@@ -0,0 +1,12 @@
+$('#start-on-boot').change((e) => {
+  Emitter.fire('settings:set', {
+    key: 'startOnBoot',
+    value: $(e.currentTarget).is(':checked'),
+  });
+});
+
+if (Settings.get('startOnBoot', true)) {
```

Comment:
Default here should be `false`.  We shouldn't force the app to start on boot for all users, that's a bad plan